src/reynir_correct/config/GreynirCorrect.conf

```diff
@@ -3579,9 +3579,153 @@ Sjíti
 Súnníti
 evrópa
 sjálfstæðismaður
+samfylkingarmaður
+framsóknarmaður
+miðflokksmaður
+pírati
 Lenínismi
 Marxismi
 Darwinismi
+Heimsstyrjöld	# Fyrri/Seinni/Síðari Heimsstyrjöld
+Stríð 			# Kalda Stríðið
+Heimsveldi
+Bylting
+Uppreisn
+Meistaradeild
+Kreppa
+Endurreisn
+Jörð
+Tungl
+vesturlönd
+norðurlönd
+óskarsverðlaun 	#ekki leiðrétt, fer í 'if all(m.stofn not in stems for m in meanings)' en meanings = []
```

Comment:
Looks like 'óskarsverðlaun' is in BÍN, or at least in our word database. Therefore, it is not corrected. Should it definitely always be 'Óskarsverðlaun'?

Reply:
According to Málfarsbankinn, it should. See here: http://malfar.arnastofnun.is/grein/67061

Reply:
OK, we'll make a BÍN erratum on this on our side.